receiver/envoyalsreceiver/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/envoya
 go 1.22.0
 
 require (
-	github.com/envoyproxy/go-control-plane v0.13.1
+	github.com/envoyproxy/go-control-plane/envoy v1.32.3
```

Comment:
looks like the previous package is replaced with a new one, not sure if this is expected. Code owner @evan-bradley please review

Reply:
yeah, envoy split to three parts:
- github.com/envoyproxy/go-control-plane v0.13.4
- github.com/envoyproxy/go-control-plane/contrib v1.32.4
- github.com/envoyproxy/go-control-plane/envoy v1.32.4

should we bump to v1.32.4?

Reply:
I'll try to rebase the PR, otherwise we can wait for the next Renovate run to catch it.

Reply:
Renovate isn't seeing 1.32.4 yet, let's handle it in a follow-up.